TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/ImageViewHelper.php

```diff
@@ -115,12 +115,16 @@ public function initializeArguments()
      * @param string $preset Preset used to determine image configuration
      * @return string an <img...> html tag
      */
-    public function render(ImageInterface $image = null, $width = null, $maximumWidth = null, $height = null, $maximumHeight = null, $allowCropping = false, $allowUpScaling = false, $async = false, $preset = null)
+    public function render($image = null, $width = null, $maximumWidth = null, $height = null, $maximumHeight = null, $allowCropping = false, $allowUpScaling = false, $async = false, $preset = null)
```

Comment:
ah now i remember, I think i did it so empty strings could pass too. But we don't really need that because the fix is mainly for missing images which are ``null``.